hail/src/main/scala/is/hail/io/bgen/LoadBgen.scala

```diff
@@ -293,8 +293,8 @@ case class MatrixBGENReader(
     rowKey = Array("locus", "alleles"),
     entryType = TStruct(
       "GT" -> TCall(),
-      "GP" -> +TArray(+TFloat64()),
-      "dosage" -> +TFloat64()))
+      "GP" -> TArray(+TFloat64()),
+      "dosage" -> TFloat64()))
```

Comment:
is there a way we can abstract so that the types of these things aren't listed in three places?